src/main/java/com/minecolonies/client/render/TileEntityScarecrowRenderer.java

```diff
@@ -97,19 +89,24 @@ public void renderTileEntityAt(@NotNull ScarecrowTileEntity te, double posX, dou
 
```

Comment:
The line above this, what happens if the TE had disappeared, now that we moved the check?